cellbrowser_tools/fov_processing.py

```diff
@@ -435,14 +425,71 @@ def _generate_paths(self):
         self.atlas_dir = atlasdir
 
     def build_file_list(self):
+        # TODO:
+        # make all this data driven.
+        # either we just take the ALL channels from the images IN ORDER
+        # or we provide some kind of listing and renaming:
+        #     source file
+        #     channel index
+        #     new name
+        #     color (maybe not even needed)
+        # this is enough data to combine together many channels from many files
+
         # start with 4 channels for membrane, structure, nucleus and transmitted light
-        self.channel_names = ["OBS_Memb", "OBS_STRUCT", "OBS_DNA", "OBS_Trans"]
+        self.channel_names = ["Membrane", "Labeled structure", "DNA", "Bright field"]
         self.channel_colors = [
             _rgba255(128, 0, 128, 255),
             _rgba255(128, 128, 0, 255),
             _rgba255(0, 128, 128, 255),
             _rgba255(128, 128, 128, 255),
         ]
+        self.channel_indices = [
+            int(self.row[DataField.ChannelNumber638]),
+            int(self.row[DataField.ChannelNumberStruct]),
+            int(self.row[DataField.ChannelNumber405]),
+            int(self.row[DataField.ChannelNumberBrightfield]),
+        ]
+
```

Comment:
below here is the special case code for the cellsystems dataset which differs from the variance dataset in the types of image channels being presented.  We currently don't have a good data-driven spec for this, so some code is doing the dirty work.